tests/console/yast2_nis.pm

```diff
@@ -98,14 +98,8 @@ sub run() {
     send_key 'alt-o';
     wait_still_screen 2;
     send_key 'alt-f';    # close the dialog...
-    assert_screen([qw(nis_server_not_found ypbind_error)]);
-    if (match_has_tag 'ypbind_error') {
-        record_soft_failure 'bsc#1073281';
-        send_key 'alt-o';
-    }
-    else {
-        send_key 'alt-o';    # close it now even when config is not valid
-    }    # check error message for 'nis server not found'
+    assert_screen('nis_server_not_found');
+    send_key 'alt-o';    # close it now even when config is not valid
```

Comment:
this is resolved by documentation, so please remove the comment in this line and on top of `assert_screen` point to:
https://documentation.suse.com/sles/15-SP5/html/SLES-all/cha-security-firewall.html#sec-security-firewall-firewalld-rpcports